src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
There should not be an empty line at the start of the class definition.

Reply:
I think it is correct, the package should have an empty line followed after it